html/videoroomtest.js

```diff
@@ -64,6 +64,7 @@ var bitrateTimer = [];
 
 var doSimulcast = (getQueryStringValue("simulcast") === "yes" || getQueryStringValue("simulcast") === "true");
 var doSimulcast2 = (getQueryStringValue("simulcast2") === "yes" || getQueryStringValue("simulcast2") === "true");
+var subscriber_mode = (getQueryStringValue("subscriber-mode") === "yes")
```

Comment:
This is missing the `true` check, if you want to be consistent with the existing properties.